tests/t_server_null_server.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+launch_server() {
+    local server_name=$1
+    local server_exec=$2
+    local server_conf=$3
+    local status="${server_name}.status"
+    local pid="${server_name}.pid"
+
+    # Ensure that old status and pid files are gone
+    rm -f "${status}" "${pid}"
+
+    "${server_exec}" \
```

Comment:
Any reason you didn't add a log file here?

Reply:
No. Just an unintended omission. Added server logging.